MapboxNavigation/InstructionsCardView.swift

```diff
@@ -52,26 +52,19 @@ public class InstructionsCardView: BaseInstructionsBannerView {
     }
     
     fileprivate func prepareManeuver(_ style: InstructionsCardStyle) {
-        maneuverView.primaryColor = style.maneuverViewPrimaryColor
-        maneuverView.secondaryColor = style.maneuverViewSecondaryColor
+        maneuverView.shouldShowHighlightedColors = false
     }
     
     fileprivate func prepareDistance(_ style: InstructionsCardStyle) {
-        distanceLabel.valueTextColor = style.distanceLabelValueTextColor
-        distanceLabel.unitTextColor = style.distanceLabelUnitTextColor
-        
-        distanceLabel.valueFont = style.distanceLabelValueFont
-        distanceLabel.unitFont = style.distanceLabelUnitFont
+        distanceLabel.showHighlightedTextColor = false
     }
     
     fileprivate func preparePrimaryLabel(_ style: InstructionsCardStyle) {
-        primaryLabel.normalFont = style.primaryLabelNormalFont
```

Comment:
@avi-c, is there still a need to customize the font used in the instruction card’s labels?

Reply:
Yes, itt should still be supported if we want this to be truly flexible.